src/common/paths/query_suggestions.yml

```diff
@@ -2,19 +2,72 @@ get:
   operationId: GetQuerySuggestions
   tags:
     - Query
-  summary: Retrieve query suggestions
+  summary: Retrieve Flux query suggestions
+  description: |
+    Retrieves a list of Flux query suggestions. Each suggestion contains a
+    [Flux function](https://docs.influxdata.com/flux/v0.x/stdlib/all-functions/)
+    name and parameters.
+
+    Use this endpoint to retrieve a list of Flux query suggestions used in the
+    InfluxDB Flux Query Builder. Helper function names have an underscore (`_`)
+    prefix and shouldn't be used  in queries--for example:
+
+    - **Not recommended**: Don't use the `_sortLimit` helper function directly.
+       - **Recommended**:  Use `top(n, columns=["_value"], tables=<-)` to sort
+        on a column and keep the top n records. `top` uses the `_sortLimit` helper function.
```

Comment:
Minor nit: It's not that they shouldn't be used in queries, it's just that they aren't meant to be used directly. They actually are used by other functions as primitives. I don't know if it's worth updating this verbiage, but just a thought. In that same vein, I don't know that the "Not recommended vs Recommended" is necessary.